docs/DEVGUIDE.md

```diff
@@ -264,7 +275,7 @@ Next, we'll register a service broker with the catalog.  To do this, we'll
 create a new [`Broker`](../contrib/examples/walkthrough/ups-broker.yaml)
 resource:
 
-    $ kubectl create -f contrib/examples/walkthrough/ups-broker.yaml
+    $ ./kubectl16 create -f contrib/examples/walkthrough/ups-broker.yaml
```

Comment:
I would be fine saying you need to use 1.6 kubectl. This is bleeding edge alpha. I think making it very clear that this required is sufficient, personally